tests/7.AlgoSDK.js

```diff
@@ -1,5 +1,6 @@
 const assert = require('assert');
 const algosdk = require('../src/index');
+const { LogicSig } = require('../src/logicsig');
```

Comment:
Similar to what I said about examples, we want our tests to import the code in a way that's as close as possible to what will happen in production. Meaning that we should prefer `algosdk.LogicSig` over importing internal files.

Reply:
that makes sense